test/integration/client-side-encryption/client_side_encryption.spec.test.js

```diff
@@ -57,9 +57,17 @@ const skippedAuthTests = [
 // TODO(NODE-4006): Investigate csfle test "operation fails with maxWireVersion < 8"
 const skippedMaxWireVersionTest = 'operation fails with maxWireVersion < 8';
 
-const SKIPPED_TESTS = new Set(
-  isAuthEnabled ? skippedAuthTests.concat(skippedMaxWireVersionTest) : [skippedMaxWireVersionTest]
-);
+const SKIPPED_TESTS = new Set([
+  ...(isAuthEnabled
+    ? skippedAuthTests.concat(skippedMaxWireVersionTest)
+    : [skippedMaxWireVersionTest]),
+  // TODO(NODE-4288): Fix FLE 2 tests
```

Comment:
should we leave a comment on Durran's current fle2 spec sync PR (NODE-4251) to rebase so that we can make sure to keep things consistent?

Reply:
Linked the jira, commented on GH, will comment on JIRA in a sec. I needed to skip these to check that there wasn't something hanging the run related to these changes.